zrcs/zrc-2.md

```diff
@@ -0,0 +1,372 @@
+| ZRC | Title                            | Status | Type     | Author                             | Created (yyyy-mm-dd) | Updated (yyyy-mm-dd) |
+| --- | -------------------------------- | ------ | -------- | ---------------------------------- | -------------------- | -------------------- |
+| 1   | Standard for Fungible Tokens | Draft  | Ready | Gareth Mensah <gareth@zilliqa.com> | 2019-11-18           | 2019-11-18           |
+
+## I. What are Fungible Tokens?
+
+The Fungible Token is an open standard for creating currencies. Fungibility is the property of a good or a commodity whose individual units are essentially interchangeable, and each of its parts is indistinguishable from another part.
```

Comment:
I'm not sure if using "good" is good (not good good :wink: ). https://english.stackexchange.com/questions/421860/is-the-word-good-as-the-singular-of-goods-correct.

Maybe rephrase as "Fungibility is the property of goods or commodities whose individual units are ..."